build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -euo pipefail
+
+mkdir -p build && cd build
+
+# Configure
+cmake -DCODE_COVERAGE=ON -DCMAKE_BUILD_TYPE=Debug ..
```

Comment:
```suggestion
cmake -DCODE_COVERAGE=ON ..
```
Unnecessary.

Reply:
No. On single config generators (e.g. `make`) it is required to pass `CMAKE_BUILD_TYPE` on multi-config generators (e.g. `MSVC`) it is required to pass `--config` to `cmake --build`.

In the first case CMake already generates ready-to-use Makefiles. I'm not sure if it does even react on `--config` later one but even if it did it would need to rerun the whole configure process resulting in increased build times.